### ISSUE TITLE
Remove SDL_ttf dependency from mingw builds

### ISSUE THREAD

**Issue by AJenbo** [diasurgical/devilutionX#3037]
Currently doing so results in the zlib target missing and thus the build fails, so some adjustments are needed, see: https://github.com/diasurgical/devilutionX/pull/3019

**Comment by StephenCWills** [comment]
Argh, that's right. I couldn't find a prebuilt download for zlib when adding libpng.

**Comment by AJenbo** [comment]
The script actually does download zlib, so we just need to figure out how to add it as a target :)

**Comment by StephenCWills** [comment]
Not exactly. It's downloading the zlib source code to get the header files, but the dll was coming from SDL_ttf. The main page for the zlib project does have links to prebuilt Windows binaries, but they are built against v1.2.3. So we have to either build it ourselves or use the old version.

I never did attempt to confirm what version SDL_ttf was using. If it was already using 1.2.3, then I guess we can just keep using that version.

**Comment by StephenCWills** [comment]
According to file properties, SDL_ttf is in fact using 1.2.11.

![image](https://user-images.githubusercontent.com/9203145/136098731-2a6949d7-50d6-40b9-b6b3-70c32516703a.png)

**Comment by AJenbo** [comment]
In that case I guess it's fair to still use it for this, but maybe we should add a note and more it a bit more clear why it's being fetched.

Isn't the header already included with SDL_image?

**Comment by AJenbo** [comment]
Ok, please see if the updated description makes sens :)

**Comment by StephenCWills** [comment]
I just discovered that Debian has a package for this.
https://packages.debian.org/stable/libz-mingw-w64-dev

And here it is for Ubuntu.
https://packages.ubuntu.com/impish/libz-mingw-w64-dev
